### PR TITLE
project survey / Dental self assessment / Cecilia F 

### PR DESCRIPTION
**Pull request by Sneezan**
**The Dental self assessment** 
- By knowing about a patients' routine, desire and special requests before the visit, 
allows the dental professional to plan and execute the treatment faster and better.  

This week's project was really fun and is a homage to my old career as a dental hygienist. 
I wish my clinic had the opportunity for patients to before the visit express requests and desires. 
It would have saved a lot of time and given me the opportunity to be a better hygienist with less stress.

**View it live**
https://selfassesment.netlify.app

**Reflections** 
The project doesn't feel completely finished, 
I will continue to work on the styling and accessibility. 
Screenshot 2022-09-30 at 12 34 02
Screenshot 2022-09-30 at 12 35 09

### CODE REVIEW

File: code/src/components/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <div style={containerStyles}>
+      <div style={fillerStyles}>
+        <span style={labelStyles} />
+      </div>
+    </div>
+  );
+};
+
```
Comment (Kristin-Larsson) [inline code/src/components/ProgressBar.js:35]: Great job with the progressbar!

File: code/src/components/YourProfessional.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Hygienist from '../images/Hygienist.png'
```
Comment (Kristin-Larsson) [inline code/src/components/YourProfessional.js:2]: Nice way to import images

File: code/src/components/Form.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  }
+
+  const onRequestInputChange = (event) => {
+    setRequestInput(event.target.value)
+  }
+
+  // return question carousel
+  return (
+    <main className="main-contaniner">
+      {counter === 0 && <FirstPage onNextQuestionChange={onNextQuestionChange} />}
```
Comment (Kristin-Larsson) [inline code/src/components/Form.js:71]: <FirstPage could be moved one step down to make all counters === look the same.

**Review by Kristin-Larsson — APPROVED** [review]
Great job!  Your survey works perfect and I really like that you have combined your previous career with your future career in a useful way.

Your code is clear & easy to understand. The naming of the components made it easy to follow.  Great job with the progress bar, styling of radio buttons and selection.
I couldn't find any approvements to make, only positive feedback! Keep up the good work!

File: code/src/components/Form.js (hunk 3 above)
Comment (Sneezan) [reply code/src/components/Form.js:71]: Thank you Kristin for the feedback ⭐️
You are right about this, that would improve the readability a bit. I will look into this. Thank you for pointing it out and going through my code! :)

**Pull request closed without merging**